### PR TITLE
Add source data format test

### PR DESCRIPTION
**Pull request by kiyohome**
To test the formatting of the source data, I added a unit test that compares the header lines. The unit test has confirmed the operation in the local environment.

I've added unit testing to your GitHub Actions workflow(YAML file). However, I haven't been able to verify how the GitHub Actions workflow works.

#11

### CODE REVIEW

File: script/expected-data-format/positiveDetail.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+hogeコード,発表日,ソース,都道府県,居住地,年齢,性別,属性,県内No.,渡航・接触歴,詳細（渡航先、接触者、関与クラスター）,ID,確定診断日,発症日,"症状(0:無症状,1:軽症・中等症,2:重症(ICU、人工呼吸器),3:検査時無症状)",死亡/退院日,コメント,"転帰(1:退院,2:死亡)",転帰ソース,コメント_1,推定感染日
```
Comment (ryo-ma) [inline script/expected-data-format/positiveDetail.csv:1]: Please delete the **hoge**.

Comment (kiyohome) [reply script/expected-data-format/positiveDetail.csv]: Ooops

Comment (kiyohome) [reply script/expected-data-format/positiveDetail.csv]: Done

File: script/test_source_data_format.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import unittest;
```
Comment (ryo-ma) [inline script/test_source_data_format.py:1]: Semicolon is unnecessary.
I have fixed it in the master branch.

Comment (kiyohome) [reply script/test_source_data_format.py:1]: Thx

**Pull request merged**